Makefile

```diff
@@ -1,20 +1,20 @@
-.PHONY: build
+.PHONY: build exec run tests clean format
+
 build:
-	dune build src/effmain.exe
+	dune build src/effmain.exe $(dune_args)
 
-.PHONY: exec
 exec:
-	dune exec src/effmain.exe -- -v --colors $(args)
+	dune exec src/effmain.exe $(dune_args) -- -v --colors $(qcheck_args)
```

Comment:
Why are you using `$(dune_args)` and not `$(exec_args)` here, is it a mistake or is there a deep reason?

Reply:
I should have explained this in the commit message. My reasoning is if the args are passed to dune, name the args `dune_args`. I think that `exec_args` sounds like arguments that are passed to the executed binary, which is not the case for `dune exec`, afaiu; we have `qcheck_args` for passing args to execution. In the meantime, `tests` target calls dune two times, so I needed to differentiate those; I could have gone for `dune_build_args` and `dune_exec_args`, but they seemed verbose. 

I could probably go with universal `dune_build_args` (for dune build), `dune_exec_args` (for dune exec), and `qcheck_args` and for passing qcheck args to the runned executable.

Reply:
I'm fine with having just `$(dune_args)` and `$(qcheck_args)`, but then the `tests` target should be changed to use `$(dune_args)` instead of `$(exec_args)` for consistency (it's because of that target that I thought you wanted `$(exec_args)` on all `dune exec` calls). I think that would be the simpler approach.